java/test/jmri/jmrit/catalog/PreviewDialogTest.java

```diff
@@ -67,7 +67,12 @@ public void testPreviewDialog()  throws FileNotFoundException, IOException {
         });
         new QueueTool().waitEmpty();
         
-        JUnitUtil.pressButton(chooser, "Choose");
+        // instead of locating the activate button, which can change, via
+        //   JUnitUtil.pressButton(chooser, "Choose");
```

Comment:
Why does the button label change?  

Is it internationalized?  

Does it change when something finishes processing?   If it changes when something finishes processing, the button changing state, the time it takes for the change to occur might impact the test code that follows.  In this respect, creating a Jenny operator is like our waitfor methods.  The jemmy operator waits for graphical objects requested to appear.